package.json

```diff
@@ -19,15 +19,12 @@
     "examples": "webpack-dev-server --config examples/webpack.config.js --no-info --content-base examples/"
   },
   "license": "MIT",
-  "peerDependencies": {
-    "react": "0.12.x"
-  },
   "dependencies": {
+    "react": "0.12.x",
```

Comment:
Wasn't related to build. These new mixins do `require('react')` directly, so it seemed like the correct thing to do. That said, `peerDependencies` is still sort of mysterious to me, so could be unnecessary?